mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 2048
+
+
+class CSFDCZMovieReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CSFDCZMovieReviewSentimentClassification",
+        description="The dataset contains 30k user reviews from csfd.cz in Czech.",
+        reference="https://arxiv.org/abs/2304.01922",
+        dataset={
+            "path": "fewshot-goes-multilingual/cs_csfd-movie-reviews",
+            "revision": "dd2ede6faaea338ef6b1e2966f06808656975a23",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2002-06-28", "2020-03-13"),
+        eval_splits=["test"],
+        eval_langs=["ces-Latn"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="CC-BY-SA-4.0",
```

Comment:
Where did we get this license info from?

Reply:
This is what the metadata on HuggingFace Datasets reports: https://huggingface.co/datasets/fewshot-goes-multilingual/cs_csfd-movie-reviews

![image](https://github.com/embeddings-benchmark/mteb/assets/461491/b42389ce-db27-47e3-86d2-daae2fdf8d95)